main.go

```diff
@@ -159,8 +159,16 @@ func main() {
 		lexer = lexers.Get(config.Input)
 	}
 
-	if config.Language != "" {
-		lexer = lexers.Get(config.Language)
+	if config.Language[0] != "" && len(config.Language) == 1 {
+		lexer = lexers.Get(config.Language[0])
+	} else if len(config.Language) == 2 {
```

Comment:
As an extension of https://github.com/charmbracelet/freeze/pull/79/files#r1568934602, if we error early then we don't need to check the length because it can't possibly be greater then 2.

Reply:
Agreed

Reply:
Should be fixed now!